.eslintrc.js

```diff
@@ -213,6 +213,8 @@ module.exports = {
           ERROR,
           { selections: ["OperationDefinition", "FragmentDefinition"] },
         ],
+        // Following rule can possibly be removed after ESLint updates.
```

Comment:
For context was this rule disabled?

Reply:
We disable the same rule [in Spruce](https://github.com/evergreen-ci/spruce/blob/71b82e4f42f9bd4aaf790d22b1126e56cb92ed62/.eslintrc.js#L236-L237); when it's on fragment imports are interpreted as comments and the linter expects a space after the `#`.